OvmfPkg/Platforms/RiscVVirt/RiscVVirt.dsc

```diff
@@ -44,6 +44,13 @@
   DEFINE NETWORK_ISCSI_ENABLE     = FALSE
   DEFINE NETWORK_ALLOW_HTTP_CONNECTIONS = TRUE
 
+  #
+  # CMO support for RV. It depends on 2 factors. First support in compiler
+  # GCC:Binutils 2.39 is required. We could make it runtime detection later
+  # using FDT or feature CSR.
+  #
+  DEFINE RV_CMO_FEATURE_AVAILABLE = FALSE

```

Comment:
Thanks. Unfortunately, there is no easy way to detect the binutils capabilities at the runtime in edk2. So, we will have to move to support only toolchain versions having > 2.39 sometime soon. A generic infrastructure to read the extensions from the FDT is better so that it works for any extension.